deploy/iso/minikube-iso/package/falco-module/falco-module.mk

```diff
@@ -4,15 +4,15 @@
 #
 ########################################################################
 
-FALCO_MODULE_VERSION = 0.24.0
+FALCO_MODULE_VERSION = 0.28.0
```

Comment:
why not all the way 0.30.0 ?

Reply:
there was some internal changes in 0.30, that causes it to take longer time

https://falco.org/blog/contribution-drivers-kmod-ebpf-libraries/

Reply:
makes sense thanks for explaining

Reply:
Yes, we were working on this last evening. If I recall correctly, Anders mentioned that Falco made some more significant changes going to 0.30 than 0.28.

It looks like Falco broke out from sysdig.

Our plan was to see if 0.28 builds, and potentially update to 0.30 later. Also, the last time Falco was updated in Minikube was Aug 25, 2020.

Reply:
It might not be more than a rename of a variable, but more testing is needed